tap_redshift/__init__.py

```diff
@@ -514,15 +528,16 @@ def main_impl():
     connection = open_connection(args.config)
     db_schema = args.config.get('schema', 'public')
     db_name = args.config.get('dbname', 'dev')
+    limit = args.config.get('limit_rows_per_batch', -1)
```

Comment:
```suggestion
    limit = args.config.get('limit_rows_per_batch', None)
```
And document this parameter.